libs/designsystem/range/src/range.component.ts

```diff
@@ -6,11 +6,13 @@ import {
   forwardRef,
   Input,
   OnChanges,
+  OnInit,
   Output,
   SimpleChanges,
+  ViewChild,
 } from '@angular/core';
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
-import { IonicModule } from '@ionic/angular';
+import { IonicModule, IonRange } from '@ionic/angular';
```

Comment:
Do we need both or should `IonicModule` be replaced by `IonRange`? 🤔

Reply:
We need IonicModule for `ion-range` to be recognizable to the template. And IonRange is not standalone so we cannot import it directly, so I think it has to be this way.

![image](https://github.com/kirbydesign/designsystem/assets/42470636/69c203c4-772d-41bd-9c94-e3d9804c02fa)